dashboard/client/src/pages/layout/index.tsx

```diff
@@ -164,6 +169,15 @@ const BasicLayout = (
                 {theme === "dark" ? <NightsStay /> : <WbSunny />}
               </Tooltip>
             </IconButton>
+            <IconButton
+              href="https://docs.ray.io/en/master/ray-core/ray-dashboard.html"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <Tooltip title="Doc">
```

Comment:
Can we make this say "Documentation" or "See documentation"? I fear this might be misunderstood.